models/comments.models.js

```diff
@@ -1,14 +1,34 @@
 const db = require("../db/connection.js");
 const { selectArticlesById } = require("./articles.models.js");
+const format = require("pg-format");
 
 exports.selectCommentsForArticle = (article_id) => {
-    return Promise.all([db
-      .query(
-        `SELECT comment_id, votes, created_at, author, body, article_id FROM comments WHERE article_id=$1;`,
-        [article_id]
-      ), selectArticlesById(article_id)])
-      .then((promises) => {
-        return promises[0].rows;
-      });
-  };
-  
\ No newline at end of file
+  return Promise.all([
+    db.query(
+      `SELECT comment_id, votes, created_at, author, body, article_id FROM comments WHERE article_id=$1;`,
+      [article_id]
+    ),
```

Comment:
I think this select should be created outside of the Promise.all array - just for readability purposes :)